resources/src/sessionCookie.php

```diff
@@ -0,0 +1,41 @@
+<?php
+require 'vendor/autoload.php';
+class SessionCookie
+{
+    public $redirect = '';
+    public function __construct()
+    {
+        $this->redirect = new PhpRedirect\Redirect;
```

Comment:
replace this with Redirect once use statement is declared at start of file.

Reply:
something like this?
```php
<?php
require 'vendor/autoload.php';
use PhpRedirect\Redirect;
class SessionCookie
{
    public $redirect = '';
    public function __construct()
    {
        $this->redirect = new Redirect;
```

Reply:
yes, something like this.